static/app/views/replays/list/replaysErroneousDeadRageCards.tsx

```diff
@@ -0,0 +1,139 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types';
+import EventView from 'sentry/utils/discover/eventView';
+import useReplayList from 'sentry/utils/replays/hooks/useReplayList';
+import {useHaveSelectedProjectsSentAnyReplayEvents} from 'sentry/utils/replays/hooks/useReplayOnboarding';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {CardReplayTable} from 'sentry/views/replays/replayTable';
+import {ReplayColumn} from 'sentry/views/replays/replayTable/types';
+
+function ReplaysErroneousDeadRageCards() {
+  const location = useLocation();
+  const organization = useOrganization();
+
+  const eventViewErrors = EventView.fromNewQueryWithLocation(
+    {
+      name: '',
+      version: 2,
+      fields: [
+        'activity',
+        'duration',
+        'count_errors',
+        'id',
+        'project_id',
+        'user',
+        'finished_at',
+        'is_archived',
+        'started_at',
+      ],
+      range: '1d',
+      projects: [],
+      query: '',
+      orderby: '-count_errors',
+    },
+    location
```

Comment:
passing in the location could also be how the `cursor` is getting in here. 

The `location` value will change every time the url is updated, and the url gets updated whenever the main project-dropdown, searchbox, or main pagination is used.